wca-guidelines.md

```diff
@@ -222,3 +222,4 @@ To be more informative, each Guideline is classified using one of the following
 - H1b1++) [ADDITION] The judge may permit the competitor to continue the attempt unofficially, but the attempt must be stopped and judged first, in full accordance with the Regulations.
 - H1d+) [EXAMPLE] Example: Suppose a competitor attempts 10 cubes, stops the solve with a time of 59:57, and has two time penalties. The time of the final result is 59:57 + 2*2 = 60:01 (also see [Regulation A1a5](regulations:regulation:A1a5)).
 - H1d++) [EXAMPLE] Example: Suppose a competitor attempts 10 cubes, the judge stops the competitor at 60 minutes, and the attempt has two time penalties. The time of the final result is 60:00 + 2*2 = 60:04.
+- H1d+++) [CLARIFICATION] If a competitor accidentally applies a move to a single puzzle during the memorization phase, the puzzle may be considered unsolved instead of the whole attempt, at the discretion of the WCA Delegate.
```

Comment:
Nit: it's not quite the attempt that is unsolved
```suggestion
- H1d+++) [CLARIFICATION] If a competitor accidentally applies any moves to one or more puzzles during the memorization phase, those puzzles may be considered unsolved at the end of the attempt (instead of immediately disqualifying the entire attempt), at the discretion of the WCA Delegate.
```